docs/catalog/cloud/optimize log collection.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Optimize log collection
+
+## Description
+Sending and storing all logs from all possible sources (workloads, services, diagnostics and platform activity) can considerably increase storage and network traffic, which would impact higher costs and carbon emissions.
+
+## Solution
+
```

Comment:
Is this different from the Reduce transmitted information pattern that exists already? https://patterns.greensoftware.foundation/catalog/cloud/reduce-transmitted-data/

Should we update that pattern or do you feel this is different enough to warrant a new pattern?

Reply:
indeed, it might fit into it. The existing pattern does not mention logs in the solution / considerations ; but could be updated

Reply:
Let's make an update to that pattern instead of introducing a new pattern that is very similar. I'll close this PR for now and we can create a new one for the update.